tox.ini

```diff
@@ -10,3 +10,6 @@ envlist = py26, py27, py34, pypy
 commands = py.test
 deps =
     pytest
+    fontTools
+    robofab
```

Comment:
Because defcon doesn't specify robofab as an install dependency, when defcon tries to import libs from robofab it's failing. Currently fontParts requires defcon, which implicitly requires robofab. Not sure how to resolve this, hence the note in the second paragraph above.